action/kafkaFeed.js

```diff
@@ -17,6 +17,11 @@ function main(params) {
     var namespace = encodeURIComponent(triggerComponents[1]);
     var trigger = encodeURIComponent(triggerComponents[2]);
 
+    if (namespace === '_') {
+        whisk.error('You must supply a non-default namespace.');
```

Comment:
@csantanapr What do you think about this error message? I was thinking maybe something like "Kafka triggers cannot be created in the default namespace. <tell them what do to instead>."